Tutorials_UsingAPIs/Flow-Control-EventQueue/main.cpp

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2020 Arm Limited and affiliates.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "mbed.h"
+#include "mbed_events.h"
+
+DigitalOut led1(LED1);
+DigitalOut led2(LED2);
+
+int main()
+{
+    // creates a queue with the default size
+    EventQueue queue;
+
+    // events are simple callbacks
+    queue.call_every(1000, [] {
```

Comment:
The follow seems clearer to me (for readability):
```c++
queue.call_every(1000, [] {led1 = !led1;});
```
It is still less than 80 chars wide. But it's your call.

Reply:
I'll change it

Reply:
I just changed this to a non-lambda callback, because lambda isn't used anywhere yet in our documentation